README.md

```diff
@@ -143,3 +143,85 @@ module "teleport" {
   elb_subnets             = "${data.terraform_remote_state.static.public_lb_subnets}"
 }
 ```
+
+## teleport-server
+
+This module will deploy Teleport on an EC2 instance. The same server will run both `auth` and `proxy`. It'll also create an EIP and a Route53 record to be able to access Teleport.
+The server will use Letsencrypt to retrieve a valid certificate for the Teleport server. It'll use the DNS challenge with Route53 to validate the domain name, but in case the Route53 sub-zone is not completely setup during the first boot and Letsencrypt fails to generate a valid certificate, the server will keep retrying until it does, and in the meantime, Teleport will use a self-signed certificate for the Web UI and API.
+
+### Available variables
+
+* [`project`]: String(required): A project where this setup belongs to. Only for naming reasons.
+* [`environment`]: String(required): The environment where this setup belongs to. Only for naming reasons.
+* [`subnet_id`]: String(required): Subnet id where the EC2 instance will be deployed.
+* [`key_name`]: String(required): SSH key name for the EC2 instance.
+* [`ami_id`]: String(required): AMI id for the EC2 instance.
```

Comment:
Maybe add that you require certain things already installed on the ami. I was now looking at the cloudinit and was confused as you only configured it but no install of 3rd party applications.

Reply:
Done